modyn/tests/selector/internal/selector_strategies/test_coreset_strategy.py

```diff
@@ -471,3 +471,107 @@ def fake_super_trigger(self):
             trigger_id, *_ = strat.trigger()
             assert trigger_id == i
             assert strat.downsampling_strategy == "RemoteGradNormDownsampling"
+
+
+def test_warmup_presampling():
+    conf = get_config_tail()
+    conf.maximum_keys_in_memory = 1
+    conf.warmup_triggers = 2
+    strat = CoresetStrategy(conf, get_minimal_modyn_config(), 0)
+
+    data1 = list(range(10))
+    timestamps1 = list(range(10))
+    labels = [0] * 10
+    strat.inform_data(data1, timestamps1, labels)
+    current_data = list(strat._get_data())
+    assert len(current_data) == 10  # trigger 0 data = warmup, so no presampling
+    current_data = flatten(current_data)
+    assert set(current_data) <= set(data1)
```

Comment:
Shouldn't these two sets exactly equal to each other, as we don't select?

Reply:
yes, this was copy pasta leftover